helix-core/src/selection.rs

```diff
@@ -222,9 +222,15 @@ impl Range {
 
     // groupAt
 
+    /// Consider using `slice` instead.
```

Comment:
This comment is a bit confusing. Could you add docs to both functions and explain the difference in return type?

Reply:
Added some docs, are they ok?

Reply:
yep looks good to me 👍